privacy-policy.ftd

```diff
@@ -114,10 +118,10 @@ active: false
 -- list-item: Purpose: These Cookies are essential to provide You with services available through the Website and to enable You to use some of its features. They help to authenticate users and prevent fraudulent use of user accounts. Without these Cookies, the services that You have asked for cannot be provided, and We only use these Cookies to provide You with those services.
```

Comment:
@SuryanshShahi 80 char limit is not followed:
https://github.com/FifthTry/admint.io/blob/60b573c9bc20c739afc3b719bc8f00f079ac7cab/privacy-policy.ftd#L21